tests/Query/Filter/PriorityField.test.ts

```diff
@@ -163,10 +200,12 @@ describe('grouping by priority', () => {
     });
 
     it.each([
-        ['- [ ] a ⏫', ['Priority 1: High']],
-        ['- [ ] a 🔼', ['Priority 2: Medium']],
-        ['- [ ] a', ['Priority 3: None']],
-        ['- [ ] a 🔽', ['Priority 4: Low']],
+        ['- [ ] a 🔝', ['Priority 1: Highest']],
+        ['- [ ] a ⏫', ['Priority 2: High']],
+        ['- [ ] a 🔼', ['Priority 3: Medium']],
+        ['- [ ] a', ['Priority 4: None']],
```

Comment:
Oh interesting... here, the existing priority numbers are visible to users...

I was thinking that adding `0` for highest was only going to affect the internals of the code...

To avoid a breaking change, the new value needs to be `0` - but if you wanted the group names to begin with `1` I would be comfortable with changing the group names here by adding `1` to the underlying numeric values in the implementation of the grouping code.

In other words, I am comfortable with this change remaining, even though the internal value of `Highest` will become `0`.

Reply:
> I guess that consistency is better, so I changed it, so the Highest is 0

Thank you!